src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -0,0 +1,4621 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "Expense Resource"
+        ],
+        "summary": "Create a new expected expense",
+        "operationId": "createUsingPOST_2",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "requestUuid",
+            "in": "path",
+            "description": "The unique identifier of the Request to which the Expense is attached",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "expense",
+            "description": "expense",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Expense"
+            }
+          },
+          {
+            "name": "userId",
+            "in": "query",
+            "description": "The unique identifier of the user performing the expense creation",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/ResourceOfExpense"
+            }
+          },
+          "400": {
+            "description": "The request is invalid",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "401": {
+            "description": "Request is not authenticated"
+          },
+          "403": {
+            "description": "Principal is not allowed to perform operation"
+          },
+          "500": {
+            "description": "An unexpected error has prevented the operation",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "503": {
+            "description": "The service is currently unavailable",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/expenses/{expenseUuid}": {
+      "get": {
+        "tags": [
+          "Expense Resource"
+        ],
+        "summary": "Get an existing expected expense",
+        "operationId": "getUsingGET_2",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "expenseUuid",
+            "in": "path",
+            "description": "The unique identifier of the Expense",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "userId",
+            "in": "query",
+            "description": "The unique identifier of the user viewing the expected expense",
```

Comment:
The unique identifier of the user viewing the Expected Expense

Reply:
No change, Leave as is in the initial doc